domain/domain.go

```diff
@@ -101,6 +103,18 @@ func (do *Domain) SetLease(lease time.Duration) {
 	do.ddl.SetLease(lease)
 }
 
+// Stat returns the DDL statistic.
+func (do *Domain) Stat() (map[string]interface{}, error) {
+	m, err := do.ddl.Stat()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	m["DDL_last_reload_schema_ts"] = atomic.LoadInt64(&do.lastLeaseTS)
```

Comment:
Can unify key of m, both use lower case, like `ddl_last_reload_schema_ts `?
Or like `DDL.LastReloadSchemaTS`?